google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/admin/instance/v1/stub/InstanceAdminStubSettings.java

```diff
@@ -232,7 +233,8 @@ public static String getDefaultEndpoint() {
 
   /** Returns a builder for the default ChannelProvider for this service. */
   public static InstantiatingGrpcChannelProvider.Builder defaultGrpcTransportProviderBuilder() {
-    return InstantiatingGrpcChannelProvider.newBuilder();
+    return InstantiatingGrpcChannelProvider.newBuilder()
```

Comment:
@chingor13 this was the change you did a large combined PR for a week or so ago right? Think this one just got picked up since we merged in the last Spanner PR yesterday.

Reply:
Yeah, this was the same contents of the bulk change to set the default.

Reply:
thanks!